pycrostates/io/ch_data.py

```diff
@@ -3,10 +3,15 @@
 from typing import Any, Union
 
 import numpy as np
-from mne.io import Info
-from mne.io.pick import _picks_to_idx, pick_info
+from mne import Info, pick_info
+from mne.utils import check_version
 from numpy.typing import NDArray
 
+if check_version("mne", "1.6"):
+    from mne._fiff.pick import _picks_to_idx
+else:
+    from mne.io.pick import _picks_to_idx
```

Comment:
You already saw that, but that's the syntax I used to load correctly the functions we used. I don't think it's adding a large maintenance burden, and we can keep MNE < 1.6 and >= 1.6.
When we will need to remove it, a simple grep on `from mne._fiff` or on `check_version("mne", "1.6")` will find all occurences.